lib/taskana-core/src/main/java/pro/taskana/spi/history/api/TaskanaHistory.java

```diff
@@ -20,4 +21,14 @@
    * @param event {@link TaskanaHistoryEvent} The event to be created.
    */
   void create(TaskanaHistoryEvent event);
+
+  /**
+   * Get the details of a history event by Id.
+   *
+   * @param historyEventId the id of the history event
+   * @return the history event
+   * @throws TaskanaHistoryEventNotFoundException If the Id rfers to a not existing history event
+   */
+  TaskanaHistoryEvent getHistoryEvent(String historyEventId)
```

Comment:
I don't think that this should be part of the SPI. This would create a dependency from core to the history we don't want to have.

Reply:
Do you have a suggestion where I should put this alternatively?